### DISCUSSION TITLE
配置proxy

### DISCUSSION THREAD

**Discussion by wlane** [he3-app/he3-feedback#274] [Q&A]
请问有配置代理的地方吗，我的网络位于内部代理之后，系统级别的全局代理没有权限配置，需要对应用配置代理才能上网，否则安装完后会报Preinstall tool failed的错误。谢谢！

**Comment by wlane** [comment]
我已经看到了其他人提出的问题，目前应该是没有单独配置代理的功能的，多谢。不知道未来是不是会在新版本中增加此功能？

**Comment by zhenweiwang1990** [comment]
很好的建议，我们考虑在后续的版本中支持此功能。

**Comment by lyzhang1999** [comment]
感谢反馈，已经创建了 issues：https://github.com/he3-app/he3-feedback/issues/275 ， 我们将持续跟进。
